osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/securitygroupinterface/UpdateMgrSecurityGroupInterfaceTask.java

```diff
@@ -50,15 +54,22 @@ public UpdateMgrSecurityGroupInterfaceTask create(SecurityGroupInterface securit
 
     @Override
     public void executeTransaction(EntityManager em) throws Exception {
-        this.securityGroupInterface = em.find(SecurityGroupInterface.class,
-                this.securityGroupInterface.getId());
+        this.securityGroupInterface = em.find(SecurityGroupInterface.class, this.securityGroupInterface.getId());
 
         ManagerSecurityGroupInterfaceApi mgrApi = this.apiFactoryService
                 .createManagerSecurityGroupInterfaceApi(this.securityGroupInterface.getVirtualSystem());
         try {
-            mgrApi.updateSecurityGroupInterface(this.securityGroupInterface.getMgrSecurityGroupIntefaceId(),
-                    this.securityGroupInterface.getName(), this.securityGroupInterface.getMgrPolicyId(),
-                    this.securityGroupInterface.getTag());
+			Set<ManagerPolicyElement> managerPolicyElements = new HashSet<>();
```

Comment:
use this.securityGroupInterface.getPolicies().stream()
    .map( policy -> new ManagerPolicyElementImpl(policy.getMgrPolicyId(),
+						policy.getName(), policy.getDomain().getMgrId()) ).collect(Collectors.toSet());